src/umpire/interface/umpire_shroud.yaml

```diff
@@ -26,6 +26,7 @@ cxx_header: umpire/Umpire.hpp
 options:
   F_module_per_class: False
   C_API_case: lower
+  wrap_python: False
```

Comment:
What does this do?  Is it required by github actions in some way?

Reply:
Nope - it controls whether or not Shroud generates a Python wrapper. I was trying to avoid having to create the empty `python` directory (it didn't help though).

I left it in since it's technically correct and somewhat clearer than leaving it out.